controllers/databasecluster_controller.go

```diff
@@ -542,7 +364,7 @@ func (r *DatabaseClusterReconciler) reconcilePSMDB(ctx context.Context, req ctrl
 			return err
 		}
 
-		psmdb.Spec = defaultPSMDBSpec
+		psmdb.Spec = *r.defaultPSMDBSpec()
```

Comment:
Why do we need to return it as a reference and use it as a value?

Reply:
It saves one copy of the struct by value.